clients/python-wrapper/lakefs/object_io.py

```diff
@@ -97,7 +130,23 @@ def create(self,
                mode: Literal['x', 'xb', 'w', 'wb'] = 'wb',
                pre_sign: Optional[bool] = None,
                content_type: Optional[str] = None,
-               metadata: Optional[dict[str, str]] = None):
+               metadata: Optional[dict[str, str]] = None) -> lakefs_sdk.ObjectStats:
+        """
+        Creates a new object or overwrites an existing object
+        :param data: The contents of the object to write (can be bytes or string)
+        :param mode: Write mode:
+            'x'     - Open for exclusive creation
+            'xb'    - Open for exclusive creation in binary mode
+            'w'     - Create a new object or truncate if exists
+            'wb'    - Create or truncate in binary mode
+        :param pre_sign: (Optional) Explicitly state whether to use pre_sign mode when uploading the object.
+        If None, will be taken from pre_sign property.
+        :param content_type: (Optional) Explicitly set the object Content-Type
+        :param metadata: (Optional) User metadata
+        :return: The Stat object representing the newly created object
+        :raises
+            ObjectExistsException if object exists and mode is exclusive ('x')
+        """
         content = data
         if mode.startswith('x') and self.exists():  # Requires explicit create
```

Comment:
```suggestion
        if 'x' in mode and self.exists():  # Requires explicit create
```

Don't assume modifier error (here and other places)

Reply:
Will do although the modifier is a Literal - so it must be in the exact order